examples/app/Synced Axes.ipynb

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
@staceysv Let's figure out where to put this and how to make it "more exemplary".

This doesn't feel like it belongs in app, it's more like a capability demonstration.

I think we'll want something like
examples
  panels
    ... (some organization here, like editor panels can have their own director)
  app (should be more like full featured dashboards)

Maybe this could go under examples/panels/plot/Synced Axes?

And some other things that could be improved:
- This example does its own data generation. It does so in a "row-oriented way", meaning it generates one row at a time with python random calls. This is too slow if we want to generate an example with 1m points
- The generated data is really random so its hard to see what's happening when you zoom. If we used nicer data it'd be easier to tell what's going on.
- I think we should organize all data generation code in one place in Weave, and have examples use that code. So far we have syndata.py, and the new Monitoring example (in a branch of mine being checked in soon) that has its own data generation, but the generated data is pretty complex and realistic. Maybe we can have a syndata directory instead, move the monitoring generation in there, and try to factor that well?

@dannygoldstein for you I think just move the notebook to where we decide in this PR and then Stacey can own refactoring the synthetic data stuff.

Reply:
thanks @shawnlewis, cc @dannygoldstein — I think `examples/panels/plot/synced_axes.ipynb` sounds good!
for immediate consistency, I think `examples/layout/synced_axes.ipynb` would also be reasonable (we'll need to move stuff around & address the other comments later).
would appreciate renaming all notebooks to lowercase + no spaces so moving stuff around is easier/for consistency, and ndb :)

Reply:
done